src/packages/protoc-gen-openapiv2/update-hash.py

```
@@ -0,0 +1,166 @@
+            text=True,
+            check=True
+        )
+        return result.stdout.strip()
+    except subprocess.CalledProcessError as e:
+        print(f"Error getting hash: {e}")
+        sys.exit(1)
+
+def get_vendor_hash(version):
+    """Get the vendor hash by attempting to build with an empty hash."""
+    nix_file = Path(__file__).parent / NIX_FILE
+    
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)